governance/reviewers-maintainers.md

```diff
@@ -1,40 +1,19 @@
-Unlike other Special Interest Groups (SIGs), the O3DE Documentation and Community SIG maintains two reviewer roles requiring SIG approval: *Documentation reviewer* and *technical reviewer*. Learn how to become either, and file a [nomination issue](https://github.com/o3de/sig-docs-community/issues/new?assignees=&labels=&template=reviewer-maintainer-nomination.md&title=SIG+Reviewer%2FMaintainer+Nomination) to become an official reviewer or maintainer!
+## Documentation and Community reviewer roles
 
-For more on the Reviewer and Maintainer roles, read our [Community Membership](https://github.com/o3de/community/blob/main/community-membership.md) information. 
+The Documentation and Community SIG maintains the following reviewer pools:
```

Comment:
I suggest using the term "teams", so it more easily correlates to the idea of GitHub teams, if anyone was trying to understand that community architecture. 
```suggestion
The Documentation and Community SIG maintains the following reviewer teams:
```